errors/index.js

```diff
@@ -5,15 +5,21 @@ exports.handleRouteErrors = (request, response, next) => {
 exports.handleCustomErrors = (error, request, response, next) => {
   if (error.statusCode && error.msg) {
     response.status(error.statusCode).send({ msg: error.msg });
-  }
-  next(error);
+  } else next(error);
 };
 
 exports.handlePsqlErrors = (error, request, response, next) => {
   if (error.code === "22P02") {
     response.status(400).send({ msg: "Bad Request" });
   }
-  next(error);
+  if (error.code === "23503") {
+    //Author is no present in users
```

Comment:
Usually I'd ask you to delete comments, but in this case they're really nice and descriptive and would contribute to people's understanding of your code, so I think it's a good thing!